features/steps/fixtures.ts

```
@@ -0,0 +1,11 @@
+import { test as base, createBdd } from 'playwright-bdd';
+
+type Fixtures = {
+  // set types of your fixtures
+};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)